webrtc.html

```diff
@@ -1750,21 +1750,29 @@ <h4>Set the RTCSessionDescription</h4>
                             object, <var>transceiver</var>, to represent the <a>media
                             description</a>.</p>
                           </li>
+                          <li>
+                            <p>If the <var>description</var> is of type <code>"offer"</code>
```

Comment:
Suggest moving this paragraph before the "find a suitable transceiver" step, to make it clear that the sendEncodings may factor into the decision on whether a transceiver is suitable or not.
(I'm not sure what we should do if the existing transceiver has sendEncodings longer than this one; both truncating the list and failing to match seems to be defensible options. We should pick one.)

Reply:
Will move it. Transceivers from *addTrack* will never have *sendEncodings* longer than one, so they should already fail to match.